cabal-install/src/Distribution/Client/CmdInstall.hs

```diff
@@ -1101,6 +1103,14 @@ disableTestsBenchsByDefault configFlags =
     , configBenchmarks = Flag False <> configBenchmarks configFlags
     }
 
+-- | Disables program prefix and suffix.
```

Comment:
Could we expand on this comment a bit? Like, why are we doing this, potentially linking to the issues, etc...
Otherwise, the reasoning might be lost in the future!

Reply:
Yes, I left the explanation at the use site, but since it is used only once let’s move it at the definition site. I expanded the explanation to explain both aspects: package hash & fix for install method.